### ISSUE TITLE
Write SQL code or R code to correct the delivery_preference field

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#115]
Operationalize SQL code or R code to correct the `delivery_preference` field in the `redcap_surveys_participants` table for the Contacts project.

When allowing surveys to be sent to the participant's preference, REDCap defaults the preference to "email" even when the email value is blank and the phone number is non-blank. These values have to be corrected manually in 'Survey Distribution Tools', 'Participant List'. These corrections have to be made daily as new contacts are added each day. It is a tedious, easily forgotten task that should be eliminated. This query will do the deed for the contacts project

```sql
update redcap_surveys_participants
set delivery_preference = "SMS_INVITE_WEB"
where participant_id in (select participant_id from 
(
SELECT rcsp.participant_id as participant_id
FROM redcap_surveys_participants rcsp
inner join redcap_surveys rcs on (rcsp.survey_id = rcs.survey_id) 
inner join redcap_surveys_response rcsr on (rcsp.participant_id = rcsr.participant_id)
left join redcap_data phone on (phone.record = rcsr.record and phone.project_id = rcs.project_id)
left join redcap_data email on (email.record = rcsr.record and email.project_id = rcs.project_id)
where rcs.project_id = 26 and
phone.field_name = 'contact_method_prefer_sms' and
email.field_name = 'contact_method_prefer_email' and
phone.value is not null and
rcsp.delivery_preference is null
) as dummy
)
;
```

This needs to be turned into an automated task that runs daily at 8:55, 13:55, 18:55

**Comment by pbchase** [comment]
@ChemiKyle, @ljwoodley, I need you guys to meet this a.m. and make a plan for making this run, 3 times per day, via automation. I lieu of that, I'll need volunteers to run it at least once per day, every day, for the next year. 

This issue is _really_ important. Addressing it will assure that new contacts get their daily symptom checks. Without it, almost none of the new contacts will get them.

**Comment by pbchase** [comment]
FYI, I submitted this code as part of a REDCap enhancement request at https://community.projectredcap.org/questions/88003/could-redcap-automatically-set-the-delivery-prefer.html

**Comment by pbchase** [comment]
Addressed by PR #120
